contracts/SafeExit.sol

```diff
@@ -39,6 +39,8 @@ contract SafeExit is Module {
         designatedToken = ERC20(_designatedToken);
         circulatingSupply = CirculatingSupply(_circulatingSupply);
 
+        if (_executor != address(0)) transferOwnership(_executor);
```

Comment:
We should pass in the owner variable to setup(), rather than transferring to executor. Since it's possible that these two might be different addresses.

For example, if this module were enabled on a modifier, rather than an executor.
Or of the DAO wants some other account to hold the modules owner permissions.

Reply:
Gotcha. I will also update the scripts to receive this owner parameter too, The `dao` parameter will be changed to `executor` and an `owner` parameter will be added @auryn-macmillan